aries-site/src/pages/templates/forms.mdx

```diff
@@ -213,11 +199,40 @@ For standard forms, use an asterisk(\*) to indicate required field/controls.
   <SingleFormFieldExample />
 </Example>
 
-## Customize
+### Customize
 
 <Example
   code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/forms/CustomizeExample.js"
   figma="https://www.figma.com/file/3fkwBelW5UsCbfwdDCJkT8/HPE-Form-Templates?node-id=54%3A608"
 >
   <CustomizeExample />
 </Example>
+
+### Accessibility
+
+All form input elements must be contained within a FormField.
+
+**FormFields must include:** 
+
+- A "for" property that matches the id of its child input. If using Grommet, this is applied with the htmlFor prop.
+- A "name" property applied on both the FormField and its child input element
+- A clear, concise label
+
+## Exceptions and how to address them
+
+- A login/registration form, which typically shows two to three required fields do not need to be annotated in any way - the form is short and the necessity of completing the fields is standard practice.
+
+- For login/registration forms, apply `required={{ indicator: false }}` to the FormFields to remove the required indicator.
```

Comment:
This should be a sub-bulllet of the above.

Reply:
I think this has changed since you wrote this.